app/config/parameters.yml.dist

```diff
@@ -73,3 +73,6 @@ parameters:
 
     # pocket import
     pocket_consumer_key: xxxxxxxx
+    
+    # sites credentials
+    sites_credentials: {}
```

Comment:
Yes, for a multi user website, I think it should be better in configuration table.